scripts/enforce-typescript-files.mjs

```diff
@@ -70,15 +69,26 @@ const LEGACY_FILE_EXCEPTIONS = [
   'spec/javascript/packages/document-capture/services/upload-spec.js',
 ];
 
-const packagesWithEntrypoints = await glob('app/javascript/packages/*/package.json')
+const packagesWithEntrypoints = await Array.fromAsync(
+  glob('app/javascript/packages/*/package.json'),
+)
   .then((files) => Promise.all(files.map(async (file) => [file, await readFile(file, 'utf-8')])))
   .then((contents) => contents.map(([file, content]) => [file, JSON.parse(content)]))
   .then((manifests) => manifests.filter(([_file, manifest]) => manifest.exports || manifest.main))
   .then((manifests) => manifests.map(([file]) => dirname(file)));
 
-const jsFiles = await glob(
```

Comment:
just curious, did we find all these by searching for `await glob(` or did they turn up as type errors?

Reply:
I had searched `fast-glob` and updated references. TypeScript did come in handy when it came to converting to using `Array.fromAsync`, since the return type of Node.js's built-in `glob` is neither an array nor a promise, but instead an `AsyncIterable`.